packages/utilities/psammead-styles/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.5 | [PR#2514](https://github.com/bbc/psammead/pull/2514) Add functions to get SerifLight and SansLight |
```

Comment:
Since we're adding new exports for `psammead-styles/font-styles`, it should be a minor update. `4.1.0`.

Reply:
Resolved